docs/Content structure/Paragraphs/Class Location.md

```diff
@@ -0,0 +1,6 @@
+# Class Location
+
+This is dynamic paragraph that renders the locations class location view mode, based on url query parameter location with a valid id.
+
+### Fields
+NONE
```

Comment:
This solution is rendering the content requested directly in the paragraph, it does not use the block field.